src/CommunityToolkit.Maui/Extensions/Popup/PopupExtensions.windows.cs

```diff
@@ -0,0 +1,26 @@
+﻿using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Views;
+using Microsoft.Maui.Controls.Platform;
+using Microsoft.Maui.Platform;
+
+namespace CommunityToolkit.Maui.Extensions;
+
+/// <summary>
+/// Extension methods for <see cref="Popup"/>.
+/// </summary>
+public static partial class PopupExtensions
+{
+	static void PlatformShowPopup(Popup popup, IMauiContext mauiContext)
```

Comment:
can we name it CreatePopup and remove this if? https://github.com/CommunityToolkit/Maui/pull/290/files#diff-dee80352ab8b3f5f51f2a55578e4b93a9fb981cafa99db2bd3991edb660368cbR25